mars/dataframe/merge/concat.py

```diff
@@ -14,6 +14,10 @@
 
 import pandas as pd
 import numpy as np
+try:
+    import cudf
+except ImportError:  # pragma: no cover
+    cudf = None
```

Comment:
Use ``lazy_import``, as processes forked after ``cudaInit()`` may be problematic.

Reply:
Done.